lib/bin/supllib/include/supl_session.h

```diff
@@ -0,0 +1,317 @@
+/*
+ * Copyright (c) 2019 - 2019, Nordic Semiconductor ASA
```

Comment:
This header file defines the library API, IMO it should be distributed with the library.

Reply:
It's distributed with the library, I don't remember why we ended up with a copy here. I can remove it.

Reply:
I removed the supl_session.h from the nrf part, this file will only be distributed with the .zip package.